.readthedocs.yml

```diff
@@ -5,6 +5,12 @@ build:
   tools:
     python: "3.10"
     nodejs: "20"
+  jobs:
+    post_install:
+      # avoid the lengthy `typedoc` build inside `sphinx` for cleaner logs
```

Comment:
Will we need to reinstate it for https://github.com/jupyterlab/jupyterlab/issues/16887?

Reply:
I don't believe so. 

Like those screenshots point out, all the `.html` is already being indexed in elasticsearch by the RTD addon, so if we opt to do special things if `READTHEDOCS`, that would be taken care of without looking at a single file... just has to be in `$READTHEDOCS_OUTPUT/html` at the end of the day.

If we want to do indexing by overloading `feed` so that PRs and local builds are right, that _would_ happen inside the sphinx lifecycle, but doing so against already-generated `.html` is fine.

If we skip both, and use something like `lunr(.js|-py)` (buh), it would also work to already have the files... and potentially even the pre-generated index.